bucket/fnm.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.22.8",
+    "description": "Cross-platform Node.js version switcher",
+    "homepage": "https://github.com/Schniz/fnm",
+    "license": "GPLv3",
```

Comment:
Invalid SPDX identifier

Reply:
What is the good identifier then ?
`GPL-3.0-only` ?